### DISCUSSION TITLE
Client-Side Navigation

### DISCUSSION THREAD

**Discussion by bedribulut** [vercel/next.js#47798] [Help]
### Summary

I was following the next.js tutorial and on the [client-side navigation chapter](https://nextjs.org/learn/basics/navigate-between-pages/client-side) it is wanted to add `background: yellow` style on developer console. I've done it but in despite of I used <Link/> component to navigate between pages, when I clicked to navigate other page the browser refreshed and set the background white again. Is there any point I am missing out?

**Comment by icyJoseph** [comment]
That may happen if you have JavaScript disabled.

**Reply by bedribulut** [reply]
Actually JavaScript is enabled. May that be related with <Link /> component entegration with the browser?

**Reply by icyJoseph** [reply]
What do you mean? If you have a reproduction demo, we could get this sorted much faster :)

**Comment by bedribulut** [comment]
firts I want to apologize for taking your time because of such a simple issue. I am a newbie and self-taught so sometime such simple issues could be challengin for me.
here it is [reproduction demo](https://nextjs-sample-website-flax.vercel.app/).
But the problem is solved after deploying on vercel.
On the live version homepage, when I change `html` background in the developer console and navigate between pages, the background remains same.
Thank you.

**Reply by icyJoseph** [reply]
No worries, is this your repository? https://github.com/bedribulut/nextjs-blog? You can open those directly on Stackblitz with: 

- https://stackblitz.com/github/bedribulut/nextjs-blog?file=README.md

I don't see any issue there. I changed the background color and navigating did not unload the page (the background color stayed as I had it).

**Reply by bedribulut** [reply]
yes, it works now but when I first started the tutorial it was not. anyway hopefully it is solved. I do appreciate for your help.
